examples/llava/tests.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# make sure we are in the right directory
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd $SCRIPT_DIR
+
+#export LLAMA_CACHE="$SCRIPT_DIR/tmp"
+
+set -eu
```

Comment:
```suggestion
set -eux
```

So we can see the commands that are executed.